core/pkg/ingress/controller/launch.go

```diff
@@ -89,6 +89,10 @@ func NewIngressController(backend ingress.Controller) *GenericController {
 		forceIsolation = flags.Bool("force-namespace-isolation", false,
 			`Force namespace isolation. This flag is required to avoid the reference of secrets or 
 		configmaps located in a different namespace than the specified in the flag --watch-namespace.`)
+
+		statusUpdateOnShutdown = flags.Bool("update-status-on-shutdown", true, `Indicates if the 
+		ingress controller should NOT update the Ingress status IP/hostname when the controller 
```

Comment:
Nit: This could probably just be `should update the Ingress` ... i.e. I think "not" is wrong there.